src/app/stop-watch/stop-watch.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="container">
+    <div>{{time$ | async}}</div>
+    <div class="monitor">
+        <div class="numbers">
+            {{ 1000 * ((time$ | async) || 0) | date: 'HH:mm:ss': 'GMT-0' }}
+        </div> 
+        <div class="control">
+            <button
+                mat-raised-button
+                color="primary"
+                [disabled]="onStart$ | async"
+                (click)="startTimer()"
+            >Start</button>
+
+            <button
+                mat-raised-button
+                color="warn"
+                [disabled]="!(time$ | async)"
+                (click)="stopTimer()"
+            >Stop</button>
+
+            <button
+                mat-raised-button
+                matTooltip="Double click to pause"
+                color="accent"
+                [disabled]="!(time$ | async)"
```

Comment:
I assume it should be disabled if not running, cause if you click wait, you still have time, but it should be disabled